esmvaltool/recipes/recipe_ocean_heatcontent.yml

```diff
@@ -0,0 +1,100 @@
+  - {dataset: HadGEM2-ES, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 1980}
+  # - {dataset: HadGEM2-ES, project: CMIP5, mip: Omon, exp: piControl, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 1980}
+  
+  # CMIP6 datasets (Feb 2019: not present on JASMIN)
+  # - {dataset: CNRM-CM6-1, project: CMIP6, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
+  # - {dataset: CNRM-ESM2-1, project: CMIP6, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
+  # - {dataset: IPSL-CM6A-LR, project: CMIP6, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
+  # - {dataset: GISS-E2-1-G, project: CMIP6, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
+  # - {dataset: BCC-ESM1, project: CMIP6, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
+  # - {dataset: BCC-CSM2-MR, project: CMIP6, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
+
+  #####
+  # Data with Errors right now -June 2018
+  ###
+  # Problem with times
+#  - {dataset: MIROC-ESM, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+#  - {dataset: MIROC-ESM-CHEM, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+
+  # Unstructured grids
+ # - {dataset: MPI-ESM-LR, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2003}
+ # - {dataset: MPI-ESM-MR, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+#  - {dataset: ACCESS1-0, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+#  - {dataset: ACCESS1-3, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+
+# TODO: All multi-model statistics are turned off until the preprocessor order is fixed.
+
+preprocessors:
+  # --------------------------------------------------
+  # Time series preprocessors
+  # --------------------------------------------------
+  prep_timeseries_1: # For 2D fields
+    custom_order: true
+#    extract_levels:
+#      levels: [700]
+#      scheme: nearest
+#    time_average:
+#
+#    extract_volume:
+#      z_min: 0
+#      z_max: 700
+#    average_volume:
+#      coordz: depth
+#      coord1: longitude
+#      coord2: latitude
+#    multi_model_statistics:
+#      span: overlap
+#      statistics: [mean ]
+
+diagnostics:
+
+  # --------------------------------------------------
+  # Time series diagnostics
+  # --------------------------------------------------
+
+  diag_timeseries_2:
+    description: Global Ocean Depth Layers mean timeseries
+    variables:
+      ohc: # Temperature ocean 3D
```

Comment:
Please simplify the datasets section by moving key-value pairs that are the same for every dataset here

Reply:
Done, thanks for the hint